Update copilot-instructions.md (#14206)
sveltejs/kit: .github/copilot-instructions.md

4. `cd packages/kit && pnpm test:unit` - Run unit tests
5. **For changes to @sveltejs/kit**: `cd packages/kit && pnpm prepublishOnly` - Generate types

#### Add changeset once you're done

> Run `npx changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`

Reviewer: Should it be using pnpm instead? The instructions mention pnpm for all the other commands.
```suggestion
Run `pnpm changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`
```

> Run `npx changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`

Reviewer: ```suggestion
Run `npx changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`. Always prefix the message with either `fix`, `feat`, `breaking`, or `chore`.
```
Wonder if it's better to give it a list of options (and maybe explain when to use which) but I think we'll probably only use this for fixes.

> Run `pnpm changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message, e.g. `fix: run load functions in order`

Author: ```suggestion
Run `pnpm changeset` then follow the prompts. Use this after having finished the task. Most of the time this is a patch release for `@sveltejs/kit`. Use a short and descriptive message. Always prefix the message with either `fix`, `feat`, `breaking`, or `chore` (most likely `fix` since you're mostly working on bugfixes). Example: `fix: run load functions in order`. Use the same message as the title of your pull request.
```
